config/index.js

```diff
@@ -28,13 +30,19 @@ var conf = convict({
     env: "PUBLIC_URL"
   },
   secretKeyFile: {
-    default: path.resolve(__dirname, '../config/secret-key.json')
+    format: String,
+    default: path.resolve(__dirname, '../config/secret-key.json'),
+    env: 'SECRET_KEY_FILE'
   },
   publicKeyFile: {
-    default: path.resolve(__dirname, '../config/public-key.json')
+    format: String,
+    default: path.resolve(__dirname, '../config/public-key.json'),
+    env: 'PUBLIC_KEY_FILE'
   },
   trustedJKUs: {
-    default: []
+    format: Array,
+    default: [],
+    env: 'TRUSTED_JKUS'
   },
```

Comment:
How do you set an array of TRUSTED_JKUS in env? Is it comma separated or something?